src/components/Sum.jsx

```diff
@@ -1 +1,10 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a, b }) => {
+  <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>;
+};
+
+Sum.defaultProps = {
+  a: 0,
+  b: 0,
+};
```

Comment:
This signature only works for class components. Use `{ prop = 10 , prop2 = 20 } `instead